includes/REST_API/Hotlinking_Controller.php

```diff
@@ -284,7 +293,7 @@ public function parse_url_permissions_check() {
 	public function validate_url( $value ) {
 		$url = untrailingslashit( $value );
 
-		if ( empty( $url ) ) {
+		if ( empty( $url ) || ! wp_http_validate_url( $url ) ) {
```

Comment:
Are checks for host / path still needed?

Reply:
The host one probably not anymore. But I'll check.

Reply:
Updated now